packages/web/src/common/store/upload/sagaHelpers.ts

```diff
@@ -106,7 +107,7 @@ export function* getUSDCMetadata(stream_conditions: USDCPurchaseConditions) {
   const priceWei = new BN(priceCents).mul(BN_USDC_CENT_WEI).toNumber()
   const conditionsWithMetadata: USDCPurchaseConditions = {
     usdc_purchase: {
-      ...stream_conditions.usdc_purchase,
+      ...omit(stream_conditions.usdc_purchase, 'albumTrackPrice'),
```

Comment:
honestly I think we remove this line entirely - I don't think any other fields belong here